README.md

```diff
@@ -63,7 +68,37 @@ This file should declare those resources, and any
 customization to apply to them, e.g. _add a common
 label_.
 
-![base image][imageBase]
+```
+
+base: kustomization + resources
+
+kustomization.yaml                      deployment.yaml                                                 service.yaml
++-------------------------------+       +-------------------------------------------------------+       +-----------------------------------+
+| commonLabels:                 |       | apiVersion: apps/v1                                   |       | apiVersion: v1                    |
+|   app: myapp                  |       | kind: Deployment                                      |       | kind: Service                     |
+| resources:                    |       | metadata:                                             |       | metadata:                         |
+|   - deployment.yaml           |       |   name: myapp                                         |       |   name: myapp                     |
+|   - service.yaml              |       | spec:                                                 |       | spec:                             |
+| configMapGenerator:           |       |   selector:                                           |       |   selector:                       |
+|   - name: myapp-map           |       |     matchLabels:                                      |       |     app: myapp                    |
+|     files:                    |       |       app: myapp                                      |       |   ports:                          |
+|       - env.startup.txt       |       |   template:                                           |       |     - port: <Port>                |
++-------------------------------+       |     metadata:                                         |       |       targetPort: <Target Port>   |
+                                        |       labels:                                         |       +-----------------------------------+
+                                        |         app: myapp                                    |
+                                        |     spec:                                             |
+                                        |       containers:                                     |
+                                        |         - name: myapp                                 |
+                                        |           image: <Image>                              |
```

Comment:
Can you please use example values rather than placeholders here and elsewhere? Since being template-free is a distinguishing feature for us, I'm thinking something that looks like a variable/placeholder might cause confusion.

Reply:
@KnVerey , done. Thanks.